austation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -11,3 +11,24 @@
 	reagent_state = SOLID
 	color = "#FFFFFF"
 	taste_description = "death"
+
+/datum/reagent/gasoline
+	name = "Gasoline"
+	description = "Petroleum derived fuel. Highly flammable."
+	reagent_state = LIQUID
+	color = "#FFBF00e9"
+	taste_description = "acrid"
+
+/datum/reagent/gasoline/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	. = ..()
+	if(method == TOUCH || method == VAPOR)
+		M.adjust_fire_stacks(reac_volume / 5)
+		return
+	..()
+
+/datum/reagent/gasoline/on_mob_life(mob/living/carbon/M)
+	. = ..()
+	M.adjustToxLoss(3,0)
+	..()
+	return TRUE
```

Comment:
Right so, `. = ..()` calls the parent proc and copies it's return value, then `..()` calls parent again AND THEN `return TRUE` completely overrides the original parent copy call.

this is broke lmao

Reply:
so are you going to actually suggest changes or just shittalk the code bruz